.github/workflows/builds.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             ${{ runner.os  }}-ccache-linux_ubuntu_${{ env.CCACHE_KEY_SUFFIX }}_
 
       - name: Configure
-        # setting CMAKE_CXX_STANDARD=17 is a workaround for a bug in the
+        # setting CMAKE_CXX_STANDARD=20 is a workaround for a bug in the
```

Comment:
still valid?

Reply:
Not sure, but I'll investigate this separately. For now I'm keeping it explicit in the CI config.